src/view_page.cpp

```diff
@@ -551,7 +553,26 @@ void View::DrawBracketsq(DeviceContext *dc, int x, int y1, int y2, int staffSize
 void View::DrawBrace(DeviceContext *dc, int x, int y1, int y2, int staffSize)
 {
     assert(dc);
-
+    if (m_doc->GetOptions()->m_useBraceGlyph.GetValue()) {
+        Glyph *glyph = Resources::GetGlyph(SMUFL_E000_brace);
+        if (glyph) {
```

Comment:
`assert(glyph)` would be enough since all the glyphs listed in smufl.h have to be available.

Reply:
Done